src/main/scala/com/cibo/scalastan/StanRunner.scala

```diff
@@ -93,17 +93,16 @@ protected object StanRunner {
       }
     }
 
-    private def readIterations(fileName: String): (Map[String, Int], Vector[Vector[String]]) = {
+    private def readIterations(fileName: String): (Vector[String], Vector[Vector[String]]) = {
```

Comment:
I think I'm missing your point. Do you mean returning (instead of a tuple) a `Map[String, Vector[String]]`, i.e., mapping header param to all iterations for that param? Sort of a compromise between the original 0.5.8 version and this one.

Reply:
I think that is what I was trying to say, but I think its more true to the theme of this version in terms of putting the `Map` at the top rather than at the bottom like 0.5.8 had.